src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -104,17 +105,19 @@ const checkRes = check(res, {
 
 </CodeGroup>
 
-### Monitor the load generator server
+### Monitor the load generator machine
 
 If you are running a test for the first time, it's a good idea to keep an eye on the available resources while the test is running.
-The easiest way to do so is to SSH to the server with 3 sessions:
+The easiest way to do so is to start at least 2 terminals one the machine: one to run k6, and others to monitor the CPU, memory and network usage.
 
-1. To run k6
-2. To monitor CPU and memory
-3. To monitor the network
+We recommend the following command-line tools for monitoring resources:
 
-For monitoring CPU and memory we recommend [htop](https://en.wikipedia.org/wiki/Htop).
-For monitoring network, we recommend [iftop](https://en.wikipedia.org/wiki/Iftop)
+- CPU and memory: [htop](https://en.wikipedia.org/wiki/Htop) or [nmon](https://nmon.sourceforge.net/).
+
+- Network: [iftop](https://en.wikipedia.org/wiki/Iftop) or [nmon](https://nmon.sourceforge.net/).
+
+If you prefer graphical applications, you can use your OS's system monitoring tool (e.g. GNOME System
+Monitor or Plasma System Monitor), or use a standalone tool like [SysMonTask](https://github.com/KrispyCamel4u/SysMonTask).
```

Comment:
Maybe we could put a little summary repetition of what the reader should be looking for here?

Something like:

```
As general guidelines, look for the the following:
- CPU utilization doesn't exceed 80%
- Network traffic isn't constant (?)
- Memory doesn't approach full utiliazation (?)

Reply:
I put (?) when I'm not sure.

Reply:
Let me know if 63f1f8e9 resolves this.